src/bin/pgcopydb/ld_transform.c

```diff
@@ -2570,7 +2569,7 @@ LogicalMessageValueEq(LogicalMessageValue *a, LogicalMessageValue *b)
 		return false;
 	}
 
-	if (a->isNull && b->isNull)
+	if (a->isNull)
```

Comment:
This one looks like a semantic change. I can see that it's not by reading the previous lines. I think it needs a comment though.

Reply:
Added

Reply:
The comment phrasing reads to me like a puzzle rather than a clue. I think a better rewrite of it would be: “given previous test, values are now known to be either both NULL or both NOT NULL“. What do you think?